e2e-tests/constants/examples/phone-number-examples.js

```diff
@@ -6,6 +6,8 @@ export const INVALID_PHONE_NUMBERS = {
   LANDLINE_SHORT: '020894223',
   LANDLINE_SPECIAL_CHAR: '0207257390*',
   LANDLINE_LETTER: '0207257390L',
+  MOBILE_SPECIAL_CHAR: '07906208973*',
+  TOO_SHORT: '*44',
```

Comment:
Can we group these so that `LANDLINE` is all together, `MOBILE` is all together etc